### ISSUE TITLE
HA Frontpage does not load on my iPad

### ISSUE THREAD

**Issue by haegar33** [home-assistant/frontend#6618]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

For all browsers on my iPad (Safari, iCab) the HA frontpage at http://192.x.x.x:81234 does not load. You only see the background patter (blue bar on top), no text, no login fields

## Expected behavior

proper login prompt and UI page as before

## Steps to reproduce

1. open HA frontpage on Safari or iCab with http **not** https

## Environment

HA core installation (virtual Python3 env) on my Raspberry 3. At latest version 114
iPad at iOS 11.2 with corresponding Safari version

## Problem-relevant configuration

I was suspecting the trusted network settings to be reasponsible but enable/disable login does not solve the problem

```yaml
  auth_providers:
   - type: trusted_networks
     trusted_networks:
       - 192.168.178.0/24
       - fd00::/8
     trusted_users:
       192.168.178.0/24: bdxxxxxxx.
```

**Comment by mountainsandcode** [comment]
More information is needed to reproduce this:
- Has it started happening with 0.114.0 or also with earlier versions?
- Does it work on other devices?
- etc.

Can you share screenshots?

**Comment by haegar33** [comment]
This bug already was present in V0.113. All browsers on Desktop Machines work fine (i.e. Firefox on Mac, Chrome in Windows etc.)
This is my "final" install on my new Pi3, as said with the core-installation method. I had a test install before on an old Pi2 with hass.io (i.e. the complete installer image) where this error was not present! Thus it might be there is a configuration error but as Desktop browsers are ok I would have no clue where to search for...
Attached a screenshot which is not very revealing ;)

![IMG_0514](https://user-images.githubusercontent.com/22793462/90392870-fdba1f80-e08f-11ea-9cfd-50ec692aa32f.PNG)

**Comment by bramkragten** [comment]
Can you check your browser logs?

**Comment by haegar33** [comment]
Browser log files? On an iPad???

**Comment by bramkragten** [comment]
Yes <https://www.lifewire.com/activate-the-debug-console-in-safari-445798>

**Comment by haegar33** [comment]
Interesting! Always something to learn ;)

The error message seems to be related to the login authorization (as suspected):

`Failed to set referrer policy: The value 'same-origin' is not one of 'no-referrer', 'origin', 'no-referrer-when-downgrade', or 'unsafe-url'. Defaulting to 'no-referrer'.`

and

`Unhandled Promise Rejection: SyntaxError: Unexpected token '...'. Expected a property name.`

I am not using https!
My current HA settings are login trusted for my LAN:

 >auth_providers:
>  - type: trusted_networks
>    trusted_networks:
>      - 192.168.178.0/24
>      - fd00::/8
>     trusted_users:
>     192.168.178.0/24: bdcd9xxxxxx

**Comment by bramkragten** [comment]
This is the issue:
`Unhandled Promise Rejection: SyntaxError: Unexpected token '...'. Expected a property name.`

Got a stack trace or a file and line number for that error?

**Comment by haegar33** [comment]
This is all on the console:

**Comment by haegar33** [comment]
still any idea on this? What kind of token is expected from HA even if everything is set to trusted networks and no login prompts?

**Comment by haegar33** [comment]
still any idea on this? What kind of token is expected from HA even if everything is set to trusted networks and no login prompts?

**Comment by haegar33** [comment]
still any idea on this? What kind of token is expected from HA even if everything is set to trusted networks and no login prompts?

**Comment by haegar33** [comment]
Further observations:
- Accessing http:/192.168.178.45:8123 (my local HA IP) via my iPhone works! The Phone is also on iOS V11.3
- no old ban_ip.yaml file found. Now the config is login disabled, trusted networks
- I enabled the debug log for homeassistant.component.http which shows these message when trying to access via the iPad (local IP 192.168.178.38)

`
2020-09-18 09:48:32 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/websocket to 192.168.178.38 (auth: False)
2020-09-18 09:48:39 DEBUG (MainThread) [homeassistant.components.http.auth] Authenticated 192.168.178.38 for /api/ using bearer token
2020-09-18 09:48:39 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/ to 192.168.178.38 (auth: True)
2020-09-18 10:56:05 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/websocket to 192.168.178.38 (auth: False)
2020-09-18 10:56:11 DEBUG (MainThread) [homeassistant.components.http.auth] Authenticated 192.168.178.38 for /api/ using bearer token
2020-09-18 10:56:11 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/ to 192.168.178.38 (auth: True)
2020-09-18 10:56:26 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/camera_proxy/camera.regenradar to 192.168.178.38 (auth: False)
2020-09-18 10:56:27 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/camera_proxy/camera.regenradar to 192.168.178.38 (auth: False)
2020-09-18 10:57:30 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/discovery_info to 192.168.178.38 (auth: False)
2020-09-18 10:59:34 DEBUG (MainThread) [homeassistant.components.http.view] Serving /hacsfiles/iconset.js to 192.168.178.38 (auth: False)
2020-09-18 10:59:35 DEBUG (MainThread) [homeassistant.components.http.view] Serving /hacsfiles/iconset.js to 192.168.178.38 (auth: False)
2020-09-18 10:59:37 DEBUG (MainThread) [homeassistant.components.http.view] Serving /hacsfiles/iconset.js to 192.168.178.38 (auth: False)
`

**Comment by haegar33** [comment]
The issue has not been resolved with updates until late November. Despite disabling all login- and user access settings HA still needs some "tokens" or any other hidden mechanism to simply logon to the system (I assume this is mainly due to the used Google polymer frameworks?).
As this iPad is my "working horse" for my smart home setup I had to abandon HA as my system of choice. I switched to Node-Red where no problems occur to control and display my smart-home dashboards.
Thanks for your efforts anyway. Forum members (and github)  for HA were very helpful in the last year!
